### PR TITLE
release 1.0.0 (#608)

### PR DESCRIPTION
**Pull request by adietish**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,8 +18,9 @@ It is available for install from the [JetBrains Marketplace](https://plugins.jet
  - Edit Kubernetes resource manifests and apply them to your cluster
  - View Kubernetes and OpenShift clusters in an explorer tree view
  - Tree View supports dynamic update of K8s resources
- - Allow Push and Pull Feature to keep the editor in sync with the Kubernetes resource
- - Support Creation and Deletion of new Kubernetes resources on Cluster
+ - Push and Pull Feature to keep the editor in sync with the Kubernetes resource
+ - Push multiple resources in an editor to create several resources at the time
```
Comment (mohitsuman) [inline README.md:22]:
```suggestion
 - Allows users to Push multiple resources in an editor to create several resources at once.
```

File: README.md [hunk 2]
```diff
@@ -79,7 +80,7 @@ There is a `Push` button available in the editor toolbar to make the action more
 ![editor toolbar push](images/editor-toolbar-push.png)
 
 Alternatively the editor notifies you whenever you may push your content to the cluster.
-Clicking the "Push" link in the editor notification saves your editor to the cluster.
+Clicking the `Push` link in the editor notification saves your editor to the cluster.
```
Comment (mohitsuman) [inline README.md:83]:
```suggestion
Clicking the `Push` link in the editor notification saves your file changes on to the cluster.
```

File: README.md [hunk 3]
```diff
@@ -88,16 +89,10 @@ The new resource will then appear in the resource tree that's displayed in our t
 
 #### Pull (Load) from Cluster
 
-The editor also notifies you if your resource has changed on the cluster.
-The notification informs you of a newer version that is available on the cluster and allows you to `Pull` it into your editor.
+You can `Pull` the resource from the cluster into you editor.
```
Comment (mohitsuman) [inline README.md:92]:
```suggestion
You can `Pull` the resource from the cluster into the local file opened in the editor. 
```

File: README.md [hunk 4]
```diff
@@ -88,16 +89,10 @@ The new resource will then appear in the resource tree that's displayed in our t
 
 #### Pull (Load) from Cluster
 
-The editor also notifies you if your resource has changed on the cluster.
-The notification informs you of a newer version that is available on the cluster and allows you to `Pull` it into your editor.
+You can `Pull` the resource from the cluster into you editor. 
+You click the `Pull` icon in the toolbar and replace the content of your editor with the version that exists on the cluster.
```
Comment (mohitsuman) [inline README.md:93]:
```suggestion
Clicking on the `Pull` icon in the toolbar replaces the content of your local file in the editor with the version that exists on the cluster.
```

File: src/main/resources/META-INF/plugin.xml [hunk 5]
```diff
@@ -10,6 +10,12 @@
     </description>
     <change-notes>
         <![CDATA[
+        <h3>1.0.0</h3>
+        <ul>
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/600">"unable to find valid certification path to requested target" when trying to edit a resource</a></li>
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/570">Should reduce number of editor notifications, they are detrimental for user focus when editing</a></li>
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/521">allow user to set ns/project with ctx menu</a></li>
```
Comment (mohitsuman) [inline src/main/resources/META-INF/plugin.xml:17]:
```suggestion
            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/521">Allows the user to set the Namespace/Project from Context Menu</a></li>
```

File: src/main/resources/META-INF/plugin.xml [hunk 6]
```diff
@@ -10,6 +10,12 @@
     </description>
     <change-notes>
         <![CDATA[
+        <h3>1.0.0</h3>
+        <ul>
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/600">"unable to find valid certification path to requested target" when trying to edit a resource</a></li>
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/570">Should reduce number of editor notifications, they are detrimental for user focus when editing</a></li>
```
Comment (mohitsuman) [inline src/main/resources/META-INF/plugin.xml:16]:
```suggestion
            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/570">Reduces number of editor notification messages when updating K8s resource files.</a></li>
```

File: src/main/resources/META-INF/plugin.xml [hunk 7]
```diff
@@ -10,6 +10,12 @@
     </description>
     <change-notes>
         <![CDATA[
+        <h3>1.0.0</h3>
+        <ul>
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/600">"unable to find valid certification path to requested target" when trying to edit a resource</a></li>
```
Comment (mohitsuman) [inline src/main/resources/META-INF/plugin.xml:15]:
```suggestion
            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/600">Support editing a K8s resource on a logged in cluster using private certificate </a></li>
```

**Review by mohitsuman — CHANGES_REQUESTED** [review]
Few wording changes.

File: src/main/resources/META-INF/plugin.xml (hunk 5 above)
Comment (adietish) [reply src/main/resources/META-INF/plugin.xml:17]: This is the exact wording of the issue as in github. Either change both, the issue and then the changelog, or none

File: src/main/resources/META-INF/plugin.xml (hunk 6 above)
Comment (adietish) [reply src/main/resources/META-INF/plugin.xml:16]: same here

File: src/main/resources/META-INF/plugin.xml (hunk 7 above)
Comment (adietish) [reply src/main/resources/META-INF/plugin.xml:15]: same

**Review by mohitsuman — APPROVED** [review]
LGTM

**Pull request merged**